day_two_guide/topics/adding_master_host.adoc

```diff
@@ -0,0 +1,177 @@
+.
+
+The prerequisites must be completed before running the scale-up procedure
+as the OpenShift installation playbooks do not include the prerequisites tasks, such as register the hosts, installing the
+required packages, `ssh` key exchange for `Ansible`, firewall rules, etc. For
+more information about the prerequisites, see
+https://docs.openshift.com/container-platform/latest/install_config/install/host_preparation.html[host preparation] documentation.
+
+All the tasks are to be performed on the host where the installation was done
+and include the `Ansible` hosts file.
+
+IMPORTANT: If manual changes in the *{rhocp}* environment exist, ensure the
+inventory file reflects those changes prior to the scale up procedure. This includes
+changes to the *{rhocp}* configuration files, for example, modifying the *{rhocp}*
+masters configuration file to customize the *{rhocp}* authentication provider as they may be overwritten.
+
+==== Procedure
+The latest `Ansible` playbooks must be available on the host that is going to do
+the scale up procedure (a workstation or the bastion host) by updating the `atomic-openshift-utils` package:
+
+[subs=+quotes]
+----
+# *yum update atomic-openshift-utils*
+----
+
+*Inventory File*
+
+The `Ansible` hosts file contains the `new_<host type>` group and this group
+belongs to the `OSEv3` group, where `new_<host_type>` can be `new_masters` or `new_nodes`
+
+NOTE: As *{rhocp}* masters are *{rhocp}* nodes, in case adding a new master,
+both `new_nodes` and `new_masters` groups must be added
+
+For example, to add a new master host, the `Ansible` hosts file needs to define the master
+under both the new_masters and the new_nodes inventory groups:
+
+[subs=+quotes]
+----
+... [OUTPUT ABBREVIATED] ...
+[OSEv3:children]
+masters
+nodes
+etcd
+*new_masters*
+*new_nodes*
+
+... [OUTPUT ABBREVIATED] ...
+[masters]
+master1.example.com
+master2.example.com
+
+[nodes]
+master1.example.com openshift_node_labels="{'role': 'master', 'zone': 'A'}"
+master2.example.com openshift_node_labels="{'role': 'master', 'zone': 'B'}"
+infranode1.example.com openshift_node_labels="{'role': 'infra', 'zone': 'A'}"
+infranode2.example.com openshift_node_labels="{'role': 'infra', 'zone': 'B'}"
+infranode3.example.com openshift_node_labels="{'role': 'infra', 'zone': 'C'}"
+node01.example.com openshift_node_labels="{'role':'app','zone':'A'}"
+node02.example.com openshift_node_labels="{'role':'app','zone':'B'}"
+node03.example.com openshift_node_labels="{'role':'app','zone':'C'}"
+
+[new_masters]
+*master3.example.com*
+
+[new_nodes]
+*master3.example.com openshift_node_labels="{'role': 'master', 'zone': 'C'}"*
+----
+
+The following command ensures access to the host and runs a simple command to ensure proper
+connectivity:
+
+[subs=+quotes]
+----
+*ansible new_masters -m command -a uptime*
+master3.example.com | SUCCESS | rc=0 >>
+ 16:05:21 up 53 min,  1 user,  load average: 0,00, 0,01, 0,05
+----
+
+*Scaleup Playbook*
```

Comment:
This is the 3rd way I've seen Scale up - is it hyphenated? not? or 1 word? be consistent.